### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by akanbrown**
I did "pip install libsodium" instead of "pip install libNaCl". New users might make the same mistake.

### CODE REVIEW

**Comment by vladimir-v-diaz** [comment]
We wish for users to execute `pip install tuf[tools]`. This one command will install multiple dependencies (e.g., 'pycrypto>=2.6.1', 'pynacl>=0.2.3') that the user will need for the repository tools.  

Maybe we should instead say, "... signature verification with Python bindings to the Networking and Cryptography Library" and link to their site.  PyNaCl relies on libsodium, a portable C library, but perhaps we don't need to mention that here.  libsodium is installed during PyNaCl's installation.

File: README.rst [hunk 1]
```diff
@@ -180,7 +180,7 @@ cryptographic keys, and signatures. Whereas the minimal install can only
 verify ed25519 signatures and is intended for sofware updater clients,
 ``tuf[tools]`` provides repository maintainers secure ed25519 key and
 signature verification with `PyNaCl <https://pynacl.readthedocs.io/en/latest/>`_,
-a Python binding to the Networking and Cryptography Library.
+a Python binding to the Networking and Cryptography (NaCl) library.
```
Comment (vladimir-v-diaz) [inline README.rst]: Should (NaCl) go after `library`?

Comment (akanbrown) [inline README.rst]: I think it's okay as is. That's the format they used on their website:

`PyNaCl: Python binding to the Networking and Cryptography (NaCl) library`

What do you think?

**Comment by vladimir-v-diaz** [comment]
I missed that.  This is okay with me also.  Merging!

**Pull request merged**